packages/mgt-element/src/utils/Cache.ts

```diff
@@ -351,12 +379,17 @@ export class CacheStore<T extends CacheItem> {
   /**
    * Returns the name of the parent DB that the cache store belongs to
    */
-  public getDBName() {
-    return `mgt-${this.schema.name}`;
+  public async getDBName(cacheId?: string) {
+    if (Providers.globalProvider && Providers.globalProvider.isMultiAccountSupportedAndEnabled) {
+      const id = await Providers.getCacheId();
+      return `mgt-${this.schema.name}` + `-${id}`;
+    } else {
+      return `mgt-${this.schema.name}`;
```

Comment:
We should also set an id even if a provider does not support multiple accounts. This will help avoid conflicts when we are testing with different providers.